ExampleMod/Content/DamageClasses/ExampleDamageClass.cs

```diff
@@ -1,65 +1,68 @@
 ﻿using System.Collections.Generic;
+using Terraria;
 using Terraria.ModLoader;
 
 namespace ExampleMod.Content.DamageClasses
 {
 	public class ExampleDamageClass : DamageClass
 	{
 		public override void SetStaticDefaults() {
-			// Make weapons with this damage type have a tooltip of 'X example damage'.
+			// This makes weapons with this damage type have a tooltip of 'X example damage'.
+			// TO-DO: Move to .hjson file and actually teach people how they work.
 			ClassName.SetDefault("example damage");
 		}
 
 		protected override float GetBenefitFrom(DamageClass damageClass) {
-			// Make this damage class not benefit from any otherclass stat bonuses by default, but still benefit from universal/all-class bonuses.
+			// This method lets you make your damage class benefit from other classes' stat bonuses by default, as well as universal stat bonuses.
+			// To briefly summarize the two nonstandard damage class names used by DamageClass:
+			// Default is, you guessed it, the default damage class. It doesn't scale off of any class-specific stat bonuses or universal stat bonuses.
+			// There are a number of items and projectiles that use this, such as thrown waters and the Bone Glove's bones.
+			// Generic, on the other hand, scales off of all universal stat bonuses and nothing else; it's the base damage class upon which all others that aren't Default are built.
 			if (damageClass == DamageClass.Generic)
 				return 1f;
 
-			// Now, I know you're just dyin' to get into the fun side of things, so let's have ourselves some demonstrations.
-			// Feel free to uncomment one of the below! Experiment, play with the variables a bit. See what works best for you!
+			return 0f;
+			// To explain how the return value works, it behaves like a percentage, with 0f being 0%, 1f being 100%, and so on.
```

Comment:
Shouldn't this comment block be above the 'return 0' statement, per the style guidance?

Reply:
Sounds reasonable to me.